spec.bs

```diff
@@ -2053,6 +2053,9 @@ an <a spec="turtledove">auction config</a> |auctionConfig| and a
             interest group set</a> whose
             <a spec="turtledove" for="interest group">owner</a> is
             |buyerOrigin|:
+            1. If <a spec="turtledove">check if required seller capabilities are
+                permitted</a> given |auctionConfig| and |ig| is false,
+                [=iteration/continue=].
             1. If seller capabilities don't allow this reporting, [=iteration/
```

Comment:
I think your new check probably replaces this one, so we should probably remove it.

Reply:
It doesn't. Someone can set requiredSellerCapabilities to be empty, but still ask for some metrics to be delegated.